### ISSUE TITLE
Giphurs SC at weight ExtraBlack has the same name than Regular weight

### ISSUE THREAD

**Issue by Corne2Plum3** [Corne2Plum3/Giphurs#16]
As shown in the screenshot below, the ExtraBlack version of the Small Cap (SC) variant of the font has the same name than the Regular name. 

![Screenshot_20240924_212444](https://github.com/user-attachments/assets/7a801c3e-1156-48e6-a1a9-5a3b06ed4e0d)

This isn't intended, the ExtraBlack should be is own family ("Giphurs ExtraBlack") with a single weight (which is "Regular"), like with "Normal" (non-SC) version. I don't know why it doesn't do that when building SC variant as well...

This bug causes the ExtraBlack weight of "Giphurs SC" not being able to be selected and used and could possibly in some edge case replace the Regular weight (400) of the font.

![Screenshot_20240924_212601](https://github.com/user-attachments/assets/5712f009-e68d-452e-8afd-eddbf4a099c5)

![Screenshot_20240924_212623](https://github.com/user-attachments/assets/669ecf4b-c8ae-403b-a25a-ba6d61dd07eb)

**Comment by Corne2Plum3** [comment]
Already tried this change in `sources/Giphurs.designspace` where I tried to change the font name for this specific weight, but didn't fix the bug.
```xml
<instance filename="ufo/Giphurs-ExtraBlack.ufo" name="Giphurs ExtraBlack" familyname="Giphurs ExtraBlack" stylename="Regular" stylemapfamilyname="Giphurs ExtraBlack" stylemapstylename="" >
  <location>
    <dimension name="weight" xvalue="1000"/>
  </location>
  <kerning/>
  <info/>
</instance>
```

**Comment by Corne2Plum3** [comment]
Solved with #17.
